banner.html.erb

```diff
@@ -0,0 +1,19 @@
+<style>
+  .container {
+    height: 100px;
+  }
+  .site-header {
+    background: url('<%=Inferno::Application['public_path']%>/bg-header-1920x170.png') repeat-x top #1fdbfe;
+  }
+  .image {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+  }
+</style>
+
+<div class='container'>
+  <div class="site-header">
+    <img class='image' alt="HealthIT.gov" src="<%= Inferno::Application['public_path'] %>/healthit.gov.logo.png">
```

Comment:
This will be a problem because actually putting images in the `public_path` from a test kit will be awkward.

Reply:
One quick solution is to just base-64 any images and inline them into here, so you don't need to serve any more assets from the web server.

Not ideal but it would work so that you wouldn't have to update inferno-core to look for a special directory to mount assets for banners to the web server.